src/main/java/Entities/Board.java

```diff
@@ -21,10 +21,27 @@ public Board(Player[] players, Cell[] cells, Object[][] propertyData){
             String name = (String) data[0];
             String colour = (String) data[1];
             int cost = (int) data[2];
```

Comment:
Why are we casting ints, wouldn't is be better if that data that is actually passed in are ints? As there are so many casting of ints to the different variables below.

Reply:
I think its because the propertyData 2d array is a Object array and the array is supposed to condense everything into one parameter.